scripts/test_model.py

```diff
@@ -0,0 +1,63 @@
+import json
+import click
+from collections import Counter
+from sklearn.metrics import classification_report
+from sklearn.externals import joblib
+from sklearn.metrics import f1_score, recall_score, precision_score
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader # noqa
+
+
+@click.command()
+@click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
+              default='PKSpace', help='Loader used to load dataset')
+@click.argument('dataset_dir',
+                type=click.Path(exists=True, file_okay=False, dir_okay=True,
+                                resolve_path=True))
+@click.argument('model_file',
+                type=click.Path(exists=True, file_okay=True, dir_okay=False,
+                                resolve_path=True))
+@click.option("--machine_friendly", '-f', is_flag=True,
```

Comment:
@rstevanak I don't really think the `--machine_friendly` is the right name to use here. I think it would be much better if it would be `--output` or `--format` and user could possibly choose from multiple formats like yaml/json for instance.